lib/stdlib/src/gen_statem.erl

```diff
@@ -2741,34 +2791,34 @@ single(false) -> "".
 mod(latin1) -> "";
 mod(_) -> "t".
 
-%% Call Module:format_status/2 or return a default value
-format_status(
-  Opt, PDict,
-  #params{modules = [Module | _]},
-  #state{state_data = {State,Data} = State_Data}) ->
-    case erlang:function_exported(Module, format_status, 2) of
-	true ->
-	    try Module:format_status(Opt, [PDict,State,Data])
-	    catch
-		Result -> Result;
-		_:_ ->
-		    format_status_default(
-		      Opt,
-                      {State,
-                       atom_to_list(Module) ++ ":format_status/2 crashed"})
-	    end;
-	false ->
-	    format_status_default(Opt, State_Data)
-    end.
-
-%% The default Module:format_status/3
-format_status_default(Opt, State_Data) ->
-    case Opt of
-	terminate ->
-	    State_Data;
-	_ ->
-	    [{data,[{"State",State_Data}]}]
-    end.
+%% %% Call Module:format_status/2 or return a default value
+%% format_status(
+%%   Opt, PDict,
+%%   #params{modules = [Module | _]},
+%%   #state{state_data = {State,Data} = State_Data}) ->
+%%     case erlang:function_exported(Module, format_status, 2) of
+%% 	true ->
+%% 	    try Module:format_status(Opt, [PDict,State,Data])
+%% 	    catch
+%% 		Result -> Result;
+%% 		_:_ ->
+%% 		    format_status_default(
+%% 		      Opt,
+%%                       {State,
+%%                        atom_to_list(Module) ++ ":format_status/2 crashed"})
+%% 	    end;
+%% 	false ->
+%% 	    format_status_default(Opt, State_Data)
+%%     end.
+
+%% %% The default Module:format_status/3
+%% format_status_default(Opt, State_Data) ->
+%%     case Opt of
+%% 	terminate ->
+%% 	    State_Data;
+%% 	_ ->
+%% 	    [{data,[{"State",State_Data}]}]
+%%     end.
```

Comment:
Remove this code instead of commenting it out.

Reply:
Fixed